src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -680,5 +779,91 @@ private static void GuideFlowEvents_GuidedTourNodeCreated(NodeModel createdNode,
                 CurrentExecutingGuide.NextStep(CurrentExecutingStep.Sequence);
             }
         }
+        
+        internal static void RemovePortConnector(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            var stepMainWindow = CurrentExecutingStep.MainWindow as Window;
+
+            if(currentFlow == GuideFlow.BACKWARD && enableFunction == false)
+            {
+                var buttonName = uiAutomationData.Parameters[0] as string;
+                if (buttonName == null) return;
+                var portHighlighted = uiAutomationData.Parameters[1] as string;
+                if (portHighlighted == null) return;
+                string nodeID = string.Empty;
+                try
+                {
+                    nodeID = uiAutomationData.Parameters[2] as string;
+                }
+                catch
+                {
+                    nodeID = string.Empty;
+                }
+
+                var outputPortViewModel = GetPortViewModel(stepMainWindow, nodeID, portHighlighted) as OutPortViewModel;
+                if (outputPortViewModel == null) return;
+                var userAddedConnector = outputPortViewModel.PortModel.Connectors.FirstOrDefault();
+                if(userAddedConnector != null)
+                    userAddedConnector.Delete();
+
+                CurrentExecutingStep.MainWindow.Dispatcher.Invoke(DispatcherPriority.Background, new Action(delegate { }));
+            }       
+        }
+
+        private static PortViewModel GetPortViewModel(Window stepMainWindow, string nodeID, string itemsControlName)
+        {
+
+            //Find a specific node(NodeView) in the MainWindow (DynamoView) based by ID or by the first of a type found
+            NodeView byOriginNode = null;
+            if (!string.IsNullOrEmpty(nodeID))
+                byOriginNode = GuideUtilities.FindNodeByID(stepMainWindow, nodeID);
+            else
+            {
+                byOriginNode = GetNotConnectedNode(stepMainWindow, "CoordinateSystem.ByOrigin");
```

Comment:
Same question here

Reply:
@QilongTang Now the value is in the json string and is passed as parameter:
https://github.com/DynamoDS/Dynamo/pull/12860/commits/ab9dac8beced65ebb949df6a3d59919273f40a45